.github/workflows/checks.yml

```diff
@@ -31,6 +31,7 @@ jobs:
                   git config user.email hey@posthog.com
 
             - name: Add safe redirects
+              if: ${{ github.base_ref == 'master' }} 
```

Comment:
not sure this is supposed to be here?

Reply:
I am not sure where that is coming from, definitely not from me